README.md

```diff
@@ -1,50 +1,76 @@
+- Minor feature changes that do not alter key functionality
+- Minor UI/UX adjustments that do not alter the overall design
+- Documentation updates
 
-1. **Finalizing the RFC**: The discussion period will remain open until the RFC is deemed complete. Upon completion, the PR will be closed. Note that closing the PR does not necessarily mean the RFC is approved; it signifies the end of the discussion phase.
+## Timeline
 
-## Community Participation
+The typical timeline for an RFC process, which is a minimum of 3 weeks, is as follows:
 
-We highly encourage community involvement in the RFC process. Whether you're proposing an RFC or contributing to the discussion of someone else's, your input is valuable.
+1. **Proposal Submission**: The RFC is submitted to the repository and made available for public comment.
+1. **Discussion Period**: A minimum of 2 weeks is allocated for community discussion and feedback. This period may be extended based on the complexity of the RFC or the volume of feedback received.
+1. **Revision Period**: (optional) Up to 1 week for the proposer to incorporate feedback and make necessary revisions to the RFC.
+1. **Final Decision**: Within 1 week after the revision period or discussion period, a decision will be made regarding the RFC's implementation.
 
-- **Informal Discussions**: Before submitting an RFC, you may want to discuss your idea informally. Join us on our [Discord](https://discord.com/invite/ktMAKGBnBs) for preliminary discussions and feedback from the community.
+## Decision Making
+### Handling Open Questions
 
-## Contribution Guidelines
+If open questions remain as the discussion timeline concludes:
 
-While contributing to the Meshtastic RFC repository, please adhere to our community standards and guidelines. These are in place to ensure a productive and respectful environment for collaboration.
+- The RFC proposer, in collaboration with core contributors, will assess whether additional information or revisions are needed to address these questions.
+- If consensus cannot be reached within the allotted time, the decision will be escalated to the Meshtastic administrators for review.
 
-- Be respectful and constructive in your communication.
-- Clearly articulate your proposals and feedback.
-- Engage in discussions with an open mind.
+### Approval Process
+
+For an RFC to be merged and accepted:
+
+1. **Consensus Among Core Contributors**: There must be approval by _at least_ one core contributor from each project/repository that will be impacted by the RFC during the 1 week **Final Decision** period.
+  - **Core Contributor**: A core contributor is someone who
+  - At the end of the period, if there are no "Request Changes" from a core contributor the RFC will be accepted/merged.
+1. **Changes Requested**: In the case that changes are requested by a core contributor and a consensus cannot be reached, the decision will be escalated to the Meshtastic administrators for a majority rule final vote.
```

Comment:
Not sure if we need to define admins, or point to a legal doc somewhere

Reply:
I could swear we had this documented somewhere. I know JM wanted to eventually have our .com domain host a separate website that had us all listed with bios and such.

Reply:
He asked me to get the .com stood up, I was hoping to work on that this weekend but it didn't happen. I think that could be a good spot to link to!

Reply:
I think we use admin in the sense of Discord, we've used the term project lead for the one or two people accountable for the direction of a specific project (eg: Garth for iOS). We do have a group of people who help maintain each project, approve PRs, etc. Maintainer may be a good one.

It may be good to have community managers at some point. Robert would be an excellent good example of this.

The people on the LLC are Members (this is the legal term) and each member may have a different title for how they see their own role play out. The LLC members are unpaid.

We should keep the roles of the LLC separate from the open source project. Right now, not every admin is a member of the LLC.

Reply:
Maybe this sort of guide is overdue. @ajmcquilkin  has been talking about growing the sense of so things projects can persist.